koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -639,7 +655,9 @@ def handler(self, src, target, opts=None):
                                      opts.get('scratch', False),
                                      opts.get('yum_repourls', None),
                                      opts.get('git_branch', None),
-                                     opts.get('push_url', None))
+                                     opts.get('push_url', None),
+                                     labels_override,
```

Comment:
This could be `labels=labels_override` to make it safer against future changes to arguments.